neo/io/baseio.py

```diff
@@ -111,7 +113,7 @@ def __init__(self, filename=None, **kargs):
             corelogger.addHandler(logging_handler)
 
     ######## General read/write methods #######################
-    def read(self, lazy=False, **kargs):
+    def read(self, lazy: bool = False, **kargs):
```

Comment:
It would be nice to have an annotation for the return type, but perhaps that's for a future PR

Reply:
Sam is so against typing I was trying to make small steps to start to get everyone on board. I think doing returns would be better in a second PR just to slowly move toward full typing.